chain/network/src/peer_manager/testonly.rs

```diff
@@ -108,14 +107,17 @@ impl ActorHandler {
     }
 }
 
-pub async fn start(cfg: config::NetworkConfig, chain: Arc<data::Chain>) -> ActorHandler {
+pub async fn start(
+    store: near_store::Store,
+    cfg: config::NetworkConfig,
+    chain: Arc<data::Chain>,
+) -> ActorHandler {
```

Comment:
Is it time to refactor this to a builder interface?

```rust
ActorHandler::builder(&mut rng)
  .store(create_test_store())
```

It doesn't feel great that adding a new argument for one call-site requires updating other call-sites. 

This can be refactored separately though, the PR lgtm!

Reply:
ack